src/bp-messages/classes/class-bp-messages-rest-controller.php

```diff
@@ -1565,6 +1566,14 @@ public function get_collection_params() {
 			'validate_callback' => 'rest_validate_request_arg',
 		);
 
+		$params['includes'] = array(
+			'description'       => __( 'Filter threads by recipient IDs.', 'buddypress' ),
+			'default'           => [],
```

Comment:
```suggestion
			'default'           => array(),
```

I love `[]` too, but for consistency it's better to use `array()` as everywhere else.